vllm/v1/worker/gpu_model_runner.py

```diff
@@ -3394,20 +3399,26 @@ def get_attn_backends_for_layers(
                         attn_backend,
                     )
 
-                key = attn_backend.full_cls_name()
```

Comment:
@LucasWilkinson Why didn't we add kv_cache_spec as a key to split kv cache groups to attention groups?